contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -170,11 +170,13 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
       ((gasPriceWei * s_config.fulfillmentGasPriceOverEstimationBP) / 10_000);
     /// @NOTE: Basis Points are 1/100th of 1%, divide by 10_000 to bring back to original units
 
-    uint96 juelsPerGas = _getJuelsPerGas(gasPriceWithOverestimation);
-    uint256 estimatedGasReimbursement = juelsPerGas * executionGas;
-    uint96 fees = uint96(donFee) + uint96(adminFee);
+    uint256 executionGas = s_config.gasOverheadBeforeCallback + s_config.gasOverheadAfterCallback + callbackGasLimit;
```

Comment:
You could keep this in line 162 to minimize noisy changes.

Reply:
I like the readability of having it all together